vector/src/main/java/im/vector/app/features/settings/devices/v2/othersessions/OtherSessionsActivity.kt

```diff
@@ -35,14 +37,22 @@ class OtherSessionsActivity : SimpleFragmentActivity() {
         if (isFirstCreation()) {
             addFragment(
                     container = views.container,
-                    fragmentClass = OtherSessionsFragment::class.java
+                    fragmentClass = OtherSessionsFragment::class.java,
+                    params = intent.getParcelableExtra(Mavericks.KEY_ARG)
             )
         }
     }
 
     companion object {
-        fun newIntent(context: Context): Intent {
-            return Intent(context, OtherSessionsActivity::class.java)
+        fun newIntent(
+                context: Context,
+                titleResourceId: Int,
```

Comment:
We may annotate this second argument with `@StringRes`.

Reply:
Done.